app/globals.css

```
@@ -556,8 +575,15 @@
 
   /* Root */
   .root-container {
-    @apply relative flex min-h-screen flex-1 flex-col px-4 pb-16 pt-4 xs:px-7 md:px-10 lg:px-14;
+    @apply relative flex min-h-screen flex-1 flex-col px-4 pt-4 xs:px-7 md:px-10 lg:px-14;
     @apply w-full max-w-full overflow-x-hidden;
+    padding-bottom: calc(6rem + env(safe-area-inset-bottom));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)